public/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Good call on adding a `manifest.json`. Let's move this into a seperate PR so it can be merged.